--- evdev.lst	2025-02-12 00:43:36.643935227 +0100
+++ /usr/share/X11/xkb/rules/evdev.lst	2025-02-12 00:49:36.709494192 +0100
@@ -669,2 +669,6 @@
   legacy          pl: Polish (legacy)
+  x_polish_colemak pl: Polish (€, Colemak Code)
+  x_polish_colemak_code pl: Polish (€, Colemak)
+  x_polish_qwerty pl: Polish (€, Colemak-Qwerty)
+  x_polish_qwerty_code pl: Polish (€, Colemak-Qwerty Code)
   qwertz          pl: Polish (QWERTZ)
